### PR TITLE
version and changelog update

### PR DESCRIPTION
**Pull request by CityVibes**
Version upgrade to 0.4.0
CHANGELOG update about 0.4.0 and changes within new version

### CODE REVIEW

File: .idea/gradle.xml [hunk 1]
```diff
@@ -14,13 +14,7 @@
             <option value="$PROJECT_DIR$/example_app" />
           </set>
         </option>
-        <option name="myModules">
```
Comment (jensck) [inline .idea/gradle.xml:17]: At some point we should remove most of the files in `.idea` from the repo.  I recently used this article as a guide for doing so on a work project: https://goo.gl/HPnSVd

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,10 @@
+0.4.0
+ =====
```
Comment (jensck) [inline CHANGELOG]: I'm seeing non-ascii characters sprinkled around this snippet, like before and after the `=` signs.

(If you can't see them, try pasting the string into a Python console as a string, and then `print`ing that string)

**Review by jensck — CHANGES_REQUESTED** [review]
Just need to strip those extra non-ascii chars from the changelog and you're good to go

File: CHANGELOG (hunk 2 above)
Comment (CityVibes) [reply CHANGELOG]: @jensck Made changes to CHANGELOG, could you check please.

File: .gitignore [hunk 3]
```diff
@@ -42,7 +42,19 @@ proguard/
 # mighty ban hammer of .gitignore.
 *.iml
 
-# OS X noise
-.DS_Store
```
Comment (jensck) [inline .gitignore:46]: You don't really want Mac .DS_Store files in the repo, do you? ;)

**Review by jensck — APPROVED** [review]
Probably should re-add .DS_Store  to .gitignore, one of them that looks good

File: .gitignore (hunk 3 above)
Comment (CityVibes) [reply .gitignore:46]: Did re-add .DS_Store, my bad on removing it while adding new stuff. :)

**Pull request merged**